vochain/transaction/election_tx.go

```diff
@@ -133,12 +147,12 @@ func (t *TransactionHandler) NewProcessTxCheck(vtx *vochaintx.VochainTx,
 	}
 
 	if tx.Process.Mode.PreRegister &&
-		(tx.Process.MaxCensusSize == nil || tx.Process.GetMaxCensusSize() <= 0) {
+		(tx.Process.MaxCensusSize == 0 || tx.Process.GetMaxCensusSize() <= 0) {
```

Comment:
this whole check became redundant, there's no chance at this point of the code that GetMaxCensusSize is <= 0, regardless of Mode.PreRegister